Libs/Widgets/ctkSettingsDialog.cpp

```diff
@@ -167,11 +167,11 @@ void ctkSettingsDialogPrivate::updateRestartRequiredLabel()
   bool restartRequired = !restartRequiredSettings.isEmpty();
   if (restartRequired)
     {
-    QString header = q->tr(
-      "<b style=\"color:red\">Restart required!</b><br>\n<small>"
-      "The application must be restarted to take into account "
-      "the new values of the following properties:\n");
-    QString footer = q->tr("</small>");
+    QString header = "<b style=\"color:red\">" + ctkSettingsDialog::tr("Restart required!") + "</b><br><small>";
+    header += ctkSettingsDialog::tr(
+      "The application must be restarted to take into account the new values of the following properties:"
+    );
```

Comment:
We still need the `\n` at the end of the header, I just asked for adding it outside the `tr()` function.

Could you add "\n" to the header after the `header += ctkSettingsDialog::tr("The application must be restarted...properties:");` line? Thank you!

Reply:
Ah ok I see, the update is done. Thanks.